.travis.yml

```
@@ -195,7 +195,7 @@ stages:
   - name: Unit Tests & Linters
     if: type =~ /(cron|push|pull_request)/ AND branch =~ /^master|[0-9]+-(dev|stable)$/
   - name: Integration tests
-    if: type =~ /(cron|push|pull_request)/ AND branch =~ /^master|[0-9]+-(dev|stable)$/
+    if: deactivated = false AND type =~ /(cron|push|pull_request)/ AND branch =~ /^master|[0-9]+-(dev|stable)$/
```

See https://github.com/kuzzleio/kuzzle/pull/1397#discussion_r320223844
```suggestion
    if: env(DEACTIVATED) = false AND type =~ /(cron|push|pull_request)/ AND branch =~ /^master|[0-9]+-(dev|stable)$/
```